packages/client/src/Call.ts

```diff
@@ -2000,28 +2016,33 @@ export class Call {
    * Applicable only for ringing calls.
    */
   private scheduleAutoDrop = () => {
-    clearTimeout(this.dropTimeout);
-    this.leaveCallHooks.add(
-      createSubscription(this.state.settings$, (settings) => {
```

Comment:
Previously, a subscription on call settings was set up when scheduling auto-drop. I don't see a reason to set up the subscription here.

Call settings should be up-to-date at this point; and in case call settings were somehow update mid-ringing, this subscription didn't take updated timeouts into account correctly anyway.

So, to simplify things, we are just setting up a timeout using current call settings.